src/main.js

```diff
@@ -590,6 +590,7 @@ let load = loader({
 
       monitor.count('expire-queues.done');
       monitor.stopResourceMonitoring();
+      queueservice.terminate();
```

Comment:
`queueService`?

Reply:
It's a component that interfaces with Azure queues.
https://github.com/taskcluster/taskcluster-queue/blob/master/src/queueservice.js#L72

It has a cache, and a `setInterval` to clear that cache every 25 hours.  That interval timer causes node to continue running forever.  Calling `terminate` cancels the interval.

Reply:
I meant: isn't that a typo? Need capital `S`

Reply:
Oh, haha, I misunderstood the question.  Yes, precisely.  I apparently failed to copy this change out of the pod where I was experimenting, too.

Reply:
(thank you for the explanation tho! I didn't realize that)